FCCee/Generator/EvtGen/Lb2LbTauTau.dec

```diff
@@ -0,0 +1,33 @@
+# Tauola steering options
+Define TauolaCurrentOption 0
+Define TauolaBR1 1.0
+#
+Alias         Mytau+     tau+
+Alias         Mytau-     tau-
+ChargeConj    Mytau+     Mytau-
+Alias   Lb_SIGNAL   Lambda_b0
+Alias   Lbbar_SIGNAL    anti-Lambda_b0
+ChargeConj Lb_SIGNAL  Lbbar_SIGNAL
+
+Alias      MyLambda0      Lambda0
+Alias      Myanti-Lambda0 anti-Lambda0
+ChargeConj MyLambda0      Myanti-Lambda0
+
+#
+Decay Lb_SIGNAL
+  1.000        MyLambda0    Mytau+    Mytau-         BTOSLLBALL;
```

Comment:
Got error during generation
```
EvtGen:BTOSLLBALL did not get the correct parent spin
Lb_SIGNAL -> MyLambda0 Mytau+ Mytau-  (BTOSLLBALL)
```
Could you double check the implementation of this model?